scripts/update-apk.sh

```diff
@@ -7,55 +7,44 @@ git config --global user.email "noreply+travis@fossasia.org"
 export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
 export PUBLISH_BRANCH=${PUBLISH_BRANCH:-master}
 
-if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-event-android" ] || ! [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" -o "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-event-attendee-android" ] || ! [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" -o "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
 	echo "We upload apk only for changes in development or master, and not PRs. So, let's skip this shall we ? :)"
 	exit 0
 fi
 
+./gradlew bundlePlayStoreRelease
```

Comment:
@iamareebjamal This will not work inside scripts directory, you need to cd to the parent directory, execute this and come back. https://github.com/fossasia/badge-magic-android/pull/428